sdk/formrecognizer/Azure.AI.FormRecognizer/src/Azure.AI.FormRecognizer.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>This is a template project to demonstrate how to create a package that uses code generation as well as use for testing our build and release pipelines</Description>
+    <AssemblyTitle>Azure SDK Template</AssemblyTitle>
+    <Version>1.0.2-preview.6</Version>
+    <PackageTags>Azure Template</PackageTags>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <EnableApiCompat>false</EnableApiCompat>
+    <!-- These supressions should be removed in a production library -->
```

Comment:
Do we have a tracking issue to remove these suppressions before shipping?

Reply:
Thanks for catching, Wes!

https://github.com/Azure/azure-sdk-for-net/issues/10455